api/management/commands/acquiredata.py

```diff
@@ -3,6 +3,12 @@
 
 from django.core.management.base import BaseCommand
 
+COMMIT_HASH = 'da832c5edc1ca4d3f665b038d15b19fced724f4c'
```

Comment:
Is it better to get this passed in from the command line along with the Genes one below?

Reply:
> I don't think we're storing it anywhere, so having it in the source code seems like it could be a good thing.

I think both commits should be in the source code (or neither). I have a preference that the commit gets tracked somewhere, so I think it's fine to be hardcoded like this or perhaps in some sort of config file.

Reply:
@kurtwheeler's approach works for me if it works for @dhimmel  👍

Reply:
Yep good with me.